src/app/routes/utils/regex/index.test.js

```diff
@@ -196,6 +221,33 @@ describe('frontPageManifestPath', () => {
   shouldNotMatchInvalidRoutes(invalidRoutes, frontPageManifestPath);
 });
 
+describe('homePageSwPath', () => {
+  const validRoutes = ['/news/tipohome/sw.js', '/persian/tipohome/sw.js'];
+  shouldMatchValidRoutes(validRoutes, homePageSwPath);
+
+  const invalidRoutes = [
+    '/news/articles/tipohome/sw.js',
+    '/persian/tipohome/sw',
+    '/persian/simp/tipohome/sw.js',
+  ];
+  shouldNotMatchInvalidRoutes(invalidRoutes, homePageSwPath);
+});
+
+describe('homePageManifestPath', () => {
+  const validRoutes = [
+    '/news/tipohome/manifest.json',
+    '/persian/tipohome/manifest.json',
+  ];
+  shouldMatchValidRoutes(validRoutes, homePageManifestPath);
+
+  const invalidRoutes = [
+    '/foobar/manifest.json',
+    '/foobar/manifest',
+    '/news/trad/sw.js',
```

Comment:
Let's use some invalid examples, like the `homePageSwPath` ones above

Reply:
Fixed :)